src/main/docker/Dockerfile

```diff
@@ -4,6 +4,10 @@ ENV SPRING_OUTPUT_ANSI_ENABLED=ALWAYS \
     JHIPSTER_SLEEP=0 \
     JAVA_OPTS=""
 
+RUN apk update && \
```

Comment:
Next time please do a separate PR for this, because this is a different issue

Reply:
You're right, I can move this to another PR if it makes things easier for you.